veracruz-mcu-client/policy_to_header.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
```

Comment:
So I've looked around quite a bit for previous projects, at many times it would have been great to have a C-friendly template engine to lean on, but every time I try one I quickly run into issues and the templating engine becomes more problems than it is worth.

There may be more issues, it's been a while since I've actually used a template engine, but some problems off the top of my head:
- Most templating engines aren't intended for C code (except maybe m4), the majority are intended for HTML (jinja, handlebars, etc), the resulting syntax can get difficult to read
- Template engines don't have as complete set of libraries as python does (though some like jinja get around this by just including python)
- State management can be a real problem. Template engines are great at unrolling lists and simple text processing, but when you need to take some data and build a structure based on that data, or convert to/from base64, or output data based on complex conditions at multiple points in multiple files, it can get messy real quick. Often the solution is to just prepare the data in python first beforehand.

The main thing python gives is flexibility, you don't have the risk of hitting some limitation halfway through the project and needing to rewrite your template to something else, since you can always use python to richly manipulate the code you are wanting to output.

But it could also just be that I was burned by template engines one too many times and I should give them another try. I've just gotten use to defaulting to python for code generation, which has the flexibility benefit and usually means no new languages need to be added to projects

./policy_to_header.py is also probably not the best example, since it's a bit messy and quickly thrown together. A lot of the generating defines are common and could be grouped together (.der -> byte arrays mostly)